src/createBaseForm.js

```diff
@@ -54,6 +54,8 @@ function createBaseForm(option = {}, mixins = []) {
         this.renderFields = {};
         this.domFields = {};
 
+        this.elementRendered = {};
```

Comment:
Where is the clean up after re-render?

Reply:
Cleanup added in `cleanUpUselessFields`;